README.md

```diff
@@ -30,18 +30,26 @@ From your shell or command line:
 
 ### Step 2:  Register the sample with your Azure Active Directory tenant
 
-There are two projects in this sample.  Each needs to be separately registered in your Azure AD tenant.
+There are two projects in this sample. Each needs to be separately registered in your Azure AD tenant. To register these projects, you can:
+
+- either follow the steps in the paragraphs below ([Step 2](#step-2--register-the-sample-with-your-azure-active-directory-tenant) and [Step 3](#step-3--configure-the-sample-to-use-your-azure-ad-tenant))
+- or use PowerShell scripts that:
+  - **automatically** create for you the Azure AD applications and related objects (passwords, permissions, dependencies)

```

Comment:
-- or --

- Follow [these instructions](./AppCreationScripts/AppCreationScripts.md) to run a script that automatically register the Azure AD applications and related objects (passwords, permissions, dependencies) required by this sample and modify the Visual Studio projects' configuration files with the necessary configuration.